tests/contracts/main/policy_manager/test_policy_manager.py

```diff
@@ -517,6 +517,240 @@ def test_create_revoke(testerchain, escrow, policy_manager):
     testerchain.wait_for_receipt(tx)
 
 
+def test_create_multiple_policies(testerchain, escrow, policy_manager):
+    creator, policy_sponsor, bad_node, node1, node2, node3, policy_owner, *everyone_else = testerchain.client.accounts
+
+    rate = 20
+    one_period = 60 * 60
+    number_of_periods = 10
+    value = rate * number_of_periods
+    default_fee_delta = policy_manager.functions.DEFAULT_FEE_DELTA().call()
+
+    policy_sponsor_balance = testerchain.client.get_balance(policy_sponsor)
+    policy_created_log = policy_manager.events.PolicyCreated.createFilter(fromBlock='latest')
+
+    # Check registered nodes
+    assert 0 < policy_manager.functions.nodes(node1).call()[PREVIOUS_FEE_PERIOD_FIELD]
+    assert 0 < policy_manager.functions.nodes(node2).call()[PREVIOUS_FEE_PERIOD_FIELD]
+    assert 0 < policy_manager.functions.nodes(node3).call()[PREVIOUS_FEE_PERIOD_FIELD]
+    assert 0 == policy_manager.functions.nodes(bad_node).call()[PREVIOUS_FEE_PERIOD_FIELD]
+    current_timestamp = testerchain.w3.eth.getBlock('latest').timestamp
+    end_timestamp = current_timestamp + (number_of_periods - 1) * one_period
+
+    policy_id_1 = os.urandom(POLICY_ID_LENGTH)
+    policy_id_2 = os.urandom(POLICY_ID_LENGTH)
+    policies = [policy_id_1, policy_id_2]
+
+    # Try to create policy for bad (unregistered) node
+    with pytest.raises((TransactionFailed, ValueError)):
+        tx = policy_manager.functions.createPolicies(policies, policy_sponsor, end_timestamp, [bad_node])\
+            .transact({'from': policy_sponsor, 'value': 2 * value})
+        testerchain.wait_for_receipt(tx)
+    with pytest.raises((TransactionFailed, ValueError)):
+        tx = policy_manager.functions.createPolicies(policies, policy_sponsor, end_timestamp, [node1, bad_node])\
+            .transact({'from': policy_sponsor, 'value': 2 * value})
+        testerchain.wait_for_receipt(tx)
+
+    # Try to create policy with no ETH
+    with pytest.raises((TransactionFailed, ValueError)):
+        tx = policy_manager.functions.createPolicies(policies, policy_sponsor, end_timestamp, [node1])\
+            .transact({'from': policy_sponsor})
+        testerchain.wait_for_receipt(tx)
+
+    # Can't create policy using timestamp from the past
+    with pytest.raises((TransactionFailed, ValueError)):
+        tx = policy_manager.functions.createPolicies(policies, policy_sponsor, current_timestamp - 1, [node1])\
+            .transact({'from': policy_sponsor, 'value': 2 * value})
+        testerchain.wait_for_receipt(tx)
+
+    # Can't create two policies with the same id
+    with pytest.raises((TransactionFailed, ValueError)):
+        tx = policy_manager.functions.createPolicies([policy_id_1, policy_id_1], policy_sponsor, end_timestamp, [node1]) \
+            .transact({'from': policy_sponsor, 'value': 2 * value, 'gas_price': 0})
```

Comment:
Does `gas_price` work? AFAIK, it should be `gasPrice`

Reply:
Checked again: yeah, `gas_price` works, and `gasPrice` too. `gasPrice` on web3 level transformed to `gas_price` for eth_tester level. I'd prefer leave as is (`gas_price`)

Reply:
The (good?) news is that the newest version of web3.py uses underscored names for all RPC endpoints.